packages/fx-core/src/plugins/resource/apim/managers/apimManager.ts

```diff
@@ -39,6 +39,9 @@ export class ApimManager {
     appName: string
   ): Promise<void> {
     const apimService: ApimService = await this.lazyApimService.getValue();
+
+    await apimService.ensureResourceProvider();
```

Comment:
Any notification/log to user when we help them to add APIM RP to the subscription?

Reply:
The logs are in ApimService#execute() func.

If the user do not have permission to add resource provider, the error message will be like:
![image](https://user-images.githubusercontent.com/49138419/127977450-dba1c0a0-1337-4586-a30d-65b6025eaef6.png)